src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -450,4 +483,17 @@ private function assertValidationMessage($element, $message)
             sprintf('Product %s should be required.', $element)
         );
     }
+
+    /**
+     * @Given I want to change its taxon
```

Comment:
I think this should not be a separate step. Our scenarios should not be coupled to interface, unless they are specifically about some kind of UI. In this scenario, we simply need to change the taxon. From UI perspective we need to open a tab, true, but that should be logic of our page implementation and should be hidden both in scenario and context. So page should ensure proper tab is open from within the method that changes the main taxon.